cloudspotting/templates/cloudspotting/cloudspotting_list.html

```diff
@@ -20,7 +20,7 @@ <h2 class="text-center">{% blocktrans %}CloudSpotting List{% endblocktrans %}</h
             <h4>{{ cloudspotting }}</h4>
             <div class="media-panel">
                 <a href="{% url "cloudspotting_detail" cloudspotting.pk %}">
-                    {% with cloudspotting.image_set.image_data.primaryImage|default:cloudspotting.image_set.image_data.images.0 as image %}
+                    {% with cloudspotting.image_set.image_data.primaryImage|default:cloudspotting.image_set.image_data.images as image %}
```

Comment:
I think we need something else. The logic should be, primaryImage else first image else nothing.